org.debrief.limpet_integration/src/org/debrief/limpet_integration/data/StoreWrapper.java

```diff
@@ -384,7 +390,32 @@ public boolean hasEditor()
     @Override
     public EditorType getInfo()
     {
-      return null;
+      if (_info == null)
+      {
+        _info = new EditorType(this, getName(), getName())
+        {
+
+          @Override
+          public PropertyDescriptor[] getPropertyDescriptors()
+          {
+            return getPropertyDescriptors();
+          }
+
+          @Override
+          public String getDisplayName()
+          {
+            return getDisplayName();
+          }
+
+          @Override
+          public String getName()
+          {
+            return getName();
+          }
```

Comment:
You are right. I have removed these methods in #1453. We need ordinal methods from EditorType.